Casks/graalvm-ce-java11.rb

```diff
@@ -29,6 +29,11 @@
     Installing GraalVM CE (Java 11) in #{JVMS_DIR} requires root permissions.
     You may be asked to enter your password to proceed.
 
+    The first time you run GraalVM CE on macOS Catalina, you may get a warning
+    that "the developer cannot be verified". This check can be disabled in the
+    "Security & Privacy" preferences pane or by running the following command:
+      xattr -r -d com.apple.quarantine #{TARGET_DIR}
```

Comment:
```suggestion
      xattr -d com.apple.quarantine #{TARGET_DIR}
```

Reply:
Are you sure this does not have to run recursively?

Reply:
GraalVM isn't following the normal app bundle layout, so I'm for keeping `-r`.

Reply:
Following your advice in https://github.com/graalvm/homebrew-tap/issues/6#issuecomment-539474120, I've successfully resolved the issue back when I installed 19.3 and when I recently upgraded to 20.0, without needing the `-r`.  So it seems like a better, smaller hammer.  Then again, it's just informational here...

Reply:
If we are sure `-r` is definitely not needed, happy to remove it. But I'd probably keep it for now. Hopefully the release bundles will be properly signed sometime soon, so this all will go away.